taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/config/OutboxRestServiceConfig.java

```diff
@@ -1,13 +1,24 @@
 package pro.taskana.adapter.camunda.outbox.rest.config;
 
+import java.util.HashSet;
+import java.util.Set;
+
 import javax.ws.rs.ApplicationPath;
 import javax.ws.rs.core.Application;
 
+import pro.taskana.adapter.camunda.outbox.rest.controller.CamundaTaskEventsController;
+
 /**
  * Configures the outbox REST service.
- * @author jhe
+ *
  */
-@ApplicationPath("")
+@ApplicationPath("/taskana-outbox/")
```

Comment:
I would have expected to find this in the application.properties file of the Spring Boot example. (server.servlet.context-path=/taskana-outbox)

Reply:
Changed